### PR TITLE
Address comments of Flink Side-Input PR

### PR DESCRIPTION
**Pull request by aljoscha**
R: @mxm

### CODE REVIEW

File: runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java [hunk 1]
```diff
@@ -245,11 +245,6 @@ public StreamingViewAsMap(View.AsMap<K, V> transform) {
 
       @SuppressWarnings({"rawtypes", "unchecked"})
       KvCoder<K, V> inputCoder = (KvCoder) input.getCoder();
-      try {
-        inputCoder.getKeyCoder().verifyDeterministic();
-      } catch (Coder.NonDeterministicException e) {
-//        runner.recordViewUsesNonDeterministicKeyCoder(this);
```
Comment (mxm) [inline runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java]: Could we keep this code and just log a warning in the catch clause?

File: runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java [hunk 2]
```diff
@@ -287,11 +282,6 @@ public StreamingViewAsMultimap(View.AsMultimap<K, V> transform) {
...
```
Comment (mxm) [inline runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java]: Could we keep this code and just log a warning in the catch clause?

**Comment by mxm** [comment]
Looks good to me. Maybe logging a warning on non-deterministic coders would be helpful.

**Comment by aljoscha** [comment]
I re-added the proper warning-logging-code that is a copy of the `DataflowRunner` code. This should be good to go now, I'm just waiting for Jenkins/Travis.

**Comment by mxm** [comment]
+1

**Pull request closed without merging**